debian/ubuntu-advantage-tools.postinst

```diff
@@ -318,6 +319,28 @@ enable_periodic_license_check() {
     fi
 }
 
+disable_new_timer_if_old_timer_already_disabled() {
+    # If the user has disable the old timer (ua-messaging)
+    # We will assume that the user would also not want the
+    # new timer to be disabled as well. Therefore, we will be
+    # manually disabling the new timer here.
```

Comment:
Looks like an accidental double-negative in here 

Also minor grammatical suggestions
```suggestion
    # If the user has disabled the old timer (ua-messaging)
    # then we will assume that the user would want the
    # new timer to be disabled as well. In that case, we will
    # disable the new timer here.
```